lib/db.js

```diff
@@ -92,24 +92,45 @@ class LimitDBRedis extends EventEmitter {
     this.buckets[key] = utils.buildBucket(bucket);
   }
 
-  bucketKeyConfig(type, key) {
-    const fromCache = type.overridesCache.get(key);
+  paramsToBucketConfig(params) {
+    const bucketConfig = {};
+
+    if (params && params.configOverride) {
+      if (params.configOverride.bucketSize) {
```

Comment:
This doesn't allow for a `0` sized bucket (basically, all gets fail).

I'm not sure if the code actually works with 0 sized buckets ;), but maybe we can check for a number here instead of truthy.